support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -56,23 +56,29 @@ DomainCreator:
       UseLogarithmicMap: true
     Envelope:
       Radius: 100.0
-      RadialDistribution: Projective
-    OuterShell:
-      Radius: 1493.0
       RadialDistribution: Logarithmic
+    OuterShell:
+      Radius: 600.0
+      RadialDistribution: Linear
       OpeningAngle: 120.0
       BoundaryCondition:
         ConstraintPreservingBjorhus:
           Type: ConstraintPreservingPhysical
     UseEquiangularMap: True
     InitialRefinement:
-      ObjectAShell:   [{{ L + 2 }}, {{ L + 2 }}, {{ L + 3 }}]
-      ObjectACube:    [{{ L + 1 }}, {{ L + 1 }}, {{ L     }}]
-      ObjectBShell:   [{{ L + 2 }}, {{ L + 2 }}, {{ L + 3 }}]
-      ObjectBCube:    [{{ L + 1 }}, {{ L + 1 }}, {{ L     }}]
-      Envelope:       [{{ L + 1 }}, {{ L + 1 }}, {{ L + 2 }}]
-      OuterShell:     [{{ L + 1 }}, {{ L + 1 }}, {{ L + 4 }}]
-    InitialGridPoints: {{ P + 1 }}
+      ObjectAShell:   [{{ L }}, {{ L }}, {{ L + 1 }}]
```

Comment:
mind aligning the braces like before? e.g. `[{{ L    }}, {{ L    }}, {{ L + 1 }}]`. It's easier to read I think.

At least some tuning went into this h-refinement, I think by @nikwit? Could you take a look @nikwit? Just checking because I don't want to throw away existing tuning. Either way, @AlexCarpenter46 please add a comment how you obtained this refinement, and what you tested, so we know next time we update this.

Reply:
oh yeah, sorry about that, will align. I've done a few tests on mbot trying out various configurations where I try to keep the amount of gridpoints constant, but move around the refinement/gridpoints to see how it affects the constraints. If you want to checkout these runs they're in `/home/fs01/spec1170/Equal_Mass/Domain_Optimizations` and I can also paste a graph of the constraints from these runs.

Reply:
I'm sure these results are at least as good as what I put in which wasn't too sophisticated.